core/config/project_settings.cpp

```diff
@@ -1119,7 +1119,9 @@ Error ProjectSettings::save_custom(const String &p_path, const CustomMap &p_cust
 		}
 	}
 	project_features = _trim_to_supported_features(project_features);
+	block_changed = true;
 	set_setting("application/config/features", project_features);
+	block_changed = false;
```

Comment:
Are these `project_features` guaranteed to never change here?

i.e. Is it better to always block here, or was `WhalesState`'s approach of comparing the `String` covering more possibilities?

Reply:
They only depend on the currently used engine build, so they don't change.